core/src/test/scala/com/pingcap/tispark/partition/PartitionWriteSuite.scala

```diff
@@ -446,9 +446,9 @@ class PartitionWriteSuite extends BaseTiSparkTest {
   }
 
   test("binary type range column replace test") {
-    if (!StoreVersion.isTiKVVersionGreatEqualThanVersion(
-        this.ti.tiSession.getPDClient,
-        "v5.1.0")) {
+    if (!StoreVersion.minTiKVVersion(
+        "v5.1.0",
+        this.ti.clientSession.getTikvSession.getPDClient)) {
```

Comment:
Here should use `isTiKVVersionGreatEqualThanVersion` . And change it in `org.tikv.common.StoreVersion` to follow master.

Reply:
Don't use `minTiKVVersion`. Please check all.

Reply:
> 不要使用`minTiKVVersion`. 请检查所有。
Unfortunately, upstream does not provide this method. After checking, minTiKVVersion and isTiKVVersionGreatEqualThanVersion are the same in result